web-console/3-project-details/labels.md

```diff
@@ -0,0 +1,71 @@
+# Labels on Detail Pages
+
+## View Labels
+- Labels are currently displayed at the top of most detail pages (Deployments, Pods, Builds, Images, Storage, etc)
+- A new `Labels` section should be added to the bottom of the `Details` tab on all of these pages to allow for a more comprehensive labels area
+
+![Label 1](img/new-labels-00-copy.png)
+
+## Add Labels
+![Label 1](img/new-labels-01-copy.png)
+
+- An `Add` link should be displayed below the applied labels
+- Clicking the link will open an overlay panel with a combo box and a text field for entering label Name and Value.
```

Comment:
May be worth saying explicitly that the user can select a key from the drop down or create their own (if that's indeed what you intend). And it probably makes sense to use the same control for the value part as well. So, for example I could choose the app label and the name of an existing app that I want to make this thing part of. The values don't have to be unique.

Reply:
Oh interesting, I was thinking the values had to be unique. If they do not, then a combo box for both would make sense to me.

Reply:
All the things that are part of the application Foo would have app=foo
How about something like this:
Clicking the link will open an overlay panel with two combo boxes so that users may select from a list of existing names and values, or create new ones.

Reply:
Makes sense to me 👍